Make sure the manpage footer contains the date, not the date within
<pubdate></pubdate>.

--- a/man/XTest.xml
+++ b/man/XTest.xml
@@ -43,7 +43,7 @@ X Consortium.
 <!-- $Xorg: xtestlib.ms,v 1.3 2000/08/17 19:42:37 cpqbld Exp $ -->
  <refentryinfo>
   <productname>__vendorversion__</productname>
-  <pubdate>6 June 2007</pubdate>
+  <date>6 June 2007</date>
    <author><firstname>Kieron</firstname><surname>Drake</surname>
    <affiliation><orgname>UniSoft Ltd.</orgname></affiliation>
    </author>
